### PR TITLE
Feature/sitemap

### PR DESCRIPTION
**Pull request by Lyokolux**

### CODE REVIEW

**Comment by Lyokolux** [comment]
#13

File: static/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,342 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+      <url>
+        <loc>https://efha-ergonomic-criteria-bastien-scapin.vercel.app/en/</loc>
+        <lastmod>2022-01-18</lastmod>
+        <changefreq>monthly</changefreq>
```
Comment (johannchopin) [inline static/sitemap.xml:6]: I guess we should ignore this file 🤔

**Review by johannchopin — APPROVED** [review]
Nice one 👍  One little comment bellow

File: static/sitemap.xml (hunk 1 above)
Comment (Lyokolux) [reply static/sitemap.xml:6]: :+1:

**Pull request merged**